docs/book/installation/sylius_plus_installation.rst

```diff
@@ -15,17 +15,19 @@ Installing Sylius Plus as a plugin to a Sylius application
 +---------------+-----------------------+
 | PHP           | ^7.3                  |
 +---------------+-----------------------+
-| sylius/sylius | ^1.7                  |
+| sylius/sylius | ^1.9                  |
++---------------+-----------------------+
+| Symfony       | ^5.0                  |
```

Comment:
I guess suggestion of `5.0` is wrong as it's not supported, the lowest supported is `5.2` (till July 2021 - https://symfony.com/releases/5.2)